### ISSUE TITLE
Simple Story Browsing UI

### ISSUE THREAD

**Issue by connortbot** [squeak-today/squeak#15]
# description
Story Browsing UI

# details
The 'home' UI of Squeak is a list of stories that can be filtered according to the drop down windows. It can also be searched through with  a search bar.

Implement independently but in parallel to the story reading UI.
![browsingui](https://github.com/user-attachments/assets/d2aec03c-7225-4729-8d3d-a67370d89481)

**Comment by connortbot** [comment]
`feat/simple-browser-ui` has functioning versions using the new `news-query` and `story-query` endpoints without a search bar and some formatting is needed.
Only need a simple browser UI for now, will make the search bar a separate issue.
